CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ find_package(rosidl_default_generators REQUIRED)
 rosidl_generate_interfaces(${PROJECT_NAME}
   "action/Fibonacci.action"
   "srv/AddTwoInts.srv"
+  ADD_LINTER_TESTS
```

Comment:
Why should the example interfaces be linted? I don't think we want to recommend this usage to users.

Reply:
I don't see how the linters are relevant for any quality level?

Reply:
Linters are defined as part of the quality level of a package https://github.com/ros-infrastructure/rep/blob/e421be8db3ef6fa65b29f9325ce635afdaecbf16/rep-2004.rst#iii-a

without linters, a package could not be more than quality 3, does make sense to test code with Level 1 or Level 2 with poor quality level package?

Reply:
I am pretty sure the REP assumes linters for code the package actually writes. Maybe that needs to be clarified in the REP.